Makefile

```diff
@@ -73,50 +76,24 @@ manifests-check:
 	@echo "Check to make sure the manifests are up to date"
 	git diff --exit-code -- config
 
-###
-# Running the operator
-###
-
-# Run against the configured Kubernetes cluster in ~/.kube/config
-run: generate vet
-	go run ./cmd/manager/main.go
-
-# Install CRDs into a cluster
-install: manifests
-	kubectl apply -f config/crds
-
-# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy: manifests
-	kubectl apply -f config/crds
-	kubectl apply -f config/operators/solr_operator.yaml
-
-###
-# Docker Building & Pushing
-###
-
-# Build the base builder docker image
-# This can be a static go build or dynamic
-docker-base-build:
```

Comment:
I think the docker build tasks should remain somewhat the same... except for a few changes in ./build/Dockerfile.build.dynamic to work with new layout (no cmd, no pkg copy commands for example) and copy in place go.mod and go.sum in that Dockerfile